website/templates/project/registration_editor_templates.mako

```diff
@@ -0,0 +1,90 @@
+<!-- String Types -->
+<script type="text/html" id="string">
+  <span data-bind="template: {data: $data, name: format}"></span>
+</script>
+
+<script type="text/html" id="text">
+  <input data-bind="valueUpdate: 'keyup',
+                    value: value"
+         type="text" class="form-control" />
+</script>
+<script type="text/html" id="match">
+  <input data-bind="valueUpdate: 'keyup',
+                    value: value,
+                    attr.placeholder: match" type="text" class="form-control" />
+</script>
+<script type="text/html" id="textarea">
+  <textarea data-bind="valueUpdate: 'keyup',
```

Comment:
TODO: Why are we updating these templates on keypress? Is this functional for some undefined spec, or a legacy of some of the previous approaches to autosave? (cue speculation)